### ISSUE TITLE
Binary Cache for Nix Packages

### ISSUE THREAD

**Issue by overtoneblue** [SergioRibera/sss#3]
Was wondering if a binary cache would be implemented for sss, as it does take a bit to build, and I believe many users would benefit from a binary cache that could be utilized for faster install. Is this at all possible? I have seen other projects implement this but it's more of a nice to have than absolutely required of course, thanks for your work on this project!

**Comment by SergioRibera** [comment]
Hello, thank you very much for using my tool 😃

I don't know very well how to do that, it probably requires making a contribution to nixpkgs, but I don't know very well how to do it either, as soon as I find out how, we could have it in nixpkgs officially

**Comment by SergioRibera** [comment]
It seems to me that this PR (https://github.com/NixOS/nixpkgs/pull/302495) could be related to this issue, let's hope they do the same for sss_code

**Comment by krovuxdev** [comment]
"Hello @cenunix ! :) Thank you for your message and your interest in improving the efficiency of installing sss!

I would like to inform you that I have made a contribution to add a sss package to nixpkgs as a contributor to the project. The PR has been submitted today and is currently in the process of review and merging. I expect that once this process is complete, users will be able to benefit from a faster installation process using the binary cache.

I appreciate your patience and ongoing support as we work on improving sss.

Thank you again for your contribution to the community!

**Comment by SergioRibera** [comment]
Hey @cenunix, we already have cachix support or so it seems, everything should be working correctly, otherwise open another issue please
